doc/sphinx/reference/onedim/grid-refinement.md

```diff
@@ -0,0 +1,228 @@
+:width: 75%
+:align: center
+
+Diagram of the slope refinement criterion.
+```
+
+
+### Curve
+
+This refinement option controls the maximum curvature of the solution component between
+two adjacent grid points compared to the maximum curvature of that component across the
+entire domain. If the ratio of the curvature between two adjacent grid points is greater
+than the user-specified value, then an additional grid point is inserted between the two
+points. This criterion is used to ensure that the solution is well-resolved in regions
+where the solution has steep gradients.
+
+The specific criteria that is being enforced is:
+
+$$
+  C_j = \frac{\left| x'_{n, j+1} - x'_{n, j} \right|}
+        {\displaystyle\max_j x'_n - \displaystyle\min_j x'_n} \le C_{max}
+$$
+
+Where $ C_j$ is computed at each grid point, $ C_{max} $ is a number between
+0 and 1 that is specified by the user, $ x'_n $ is the derivative of the
+solution component being considered, and $ \displaystyle\max_j x'_n $ and
+$ \displaystyle\min_j x'_n $ are the maximum and minimum values of the
+solution component derivative across the domain. Points where $ C_j $ is greater
+than $ C_{max} $ are marked to have an additional grid point inserted to the
+right of them. Visually, the minimum and maximum global quantities are shown in the
+image below.
+
+```{figure} /_static/images/onedim/refinement_curve_metric_plot.svg
+:width: 75%
+:alt: Diagram of the curve refinement criterion.
+:align: center
+
+Diagram of the curve refinement criterion.
+```
+
+**Note:** The slope is computed using a backward difference scheme and so the estimate of
+the derivative at a grid point depends on the solution at the grid point and the
+solution at the grid point to the right.
+
+$$
+  x'_{n, j} \approx \frac{x_{n,j+1} - x_{n,j}}{z_{j+1} - z_{j}}
+$$
+
+
+### Prune
+
+This refinement option works in contrast to the other refinement options. Instead of
+adding grid points, this option removes grid points that are not needed. The pruning
+parameter helps the grid refinement algorithm identify areas where the grid may be
+overly refined. It is most useful when running multiple simulations where the boundary
+conditions change, and the location of the regions where grid refinement is needed
+changes. In this case, the prune parameter can be used to remove grid points that are
+no longer needed.
+
+You specify a pruning value $ P $ that serves as a lower-bound on
+the ratio quantities. This helps prevent unnecessary grid refinement by
+removing points that are not needed, optimizing computational efficiency without
+sacrificing the accuracy of the solution. The following criteria are used to decide
+if a point is marked for removal:
+
+$$
+  \frac{S_j}{S_{\text{max}}} < P \quad \text{or}
+  \quad \frac{C_j}{C_{\text{max}}} < P
+$$
+
+If $ \frac{S_j}{S_{max}} $, or $ \frac{C_j}{C_{max}} $ are less
+than $ P $, then the point is marked for removal.
```

Comment:
The criterion is implemented such that if either ratio is greater than or equal to $P$, the point is kept. The converse of this is that only if _both_ ratios are less than $P$ is the point marked for removal.
```suggestion
$$
  \frac{S_j}{S_{\text{max}}} < P \quad \text{and}
  \quad \frac{C_j}{C_{\text{max}}} < P
$$

If $ \frac{S_j}{S_{max}} $ and $ \frac{C_j}{C_{max}} $ are less
than $ P $, then the point is marked for removal.
```

Reply:
Good catch!